### ISSUE TITLE
Review delivering a digital-first public experience

### ISSUE THREAD

**Issue by Maroyafaied** [DOI-ONRR/nrrd#2847]
On September 22, 2023, the Office of Management and Budget (OMB) issued M-23-22, Delivering a Digital-First Public Experience.

The 21st Century Integrated Digital Experience Act, otherwise known as 21st Century IDEA, was a bipartisan act signed into law in December 2018. 

The law and policy guidance collectively establish a framework and the requirements for a digital-first public experience.

Visit [Digital.gov’s 21st Century IDEA resource page](https://digital.gov/resources/21st-century-integrated-digital-experience-act/) for more information.

**Comment by cthomasONRR** [comment]
A lot of what this memo discusses - we already covered in the USWDS research https://blog-nrrd.doi.gov/USWDS/

**Comment by eelzi-ONRR** [comment]
Overall we're doing really good with what is listed in this. The one area we really need to work on is number 8: Mobile-first design across devices.

We're less that the 50% mobile users that represent users for all govt sites, but they discuss a mobile-first experience and we have some work to do for that.

For other parts, even though we're already doing it, we can use this document for leverage if we ever need to justify why we are doing something.

For Action 1 in the reporting section - who is our digital experience delivery lead?

**Comment by Maroyafaied** [comment]
@eelzi-ONRR Thanks for reading and pointing out the mobile user experience. Also, sorry I didn't tell you earlier but your new title is "Digital Experience Delivery Lead" 🥇 lol jk

**Comment by Maroyafaied** [comment]
I've [read this pdf ](https://www.whitehouse.gov/wp-content/uploads/2023/09/M-23-22-Delivering-a-Digital-First-Public-Experience.pdf)and I agree we're doing a pretty good job already. There are a few action items we'll need to do or look into and I made some issues for them.
1. Identify Digital experience delivery lead issue #2904
2. Assess common questions and top web content for deduplication and SEO issue #2905
3. Assess top tasks for self-service optimization issue#2906
4. Label external links to non-federal websites  #2908

**Comment by cthomasONRR** [comment]
@Maroyafaied Issue #2908 can be easily done in NRRD with the content database.

**Comment by Maroyafaied** [comment]
@cthomasONRR that's great.  For anyone that hasn't reviewed this yet, I found this pdf to be [detailed and helpful ](https://www.whitehouse.gov/wp-content/uploads/2023/09/M-23-22-Delivering-a-Digital-First-Public-Experience.pdf)

**Comment by lpgoldstein** [comment]
I agree that we are doing a good job but have questions about the rest of our agency.

**Comment by AlexandracmONRR** [comment]
Agreed that we are doing well in many of these categories.

The one goal that made me pause was this:
OMB’s policy guidance (M-23-22) requires to the greatest extent practicable that agencies:
- Make forms available to the public in a digital format
- **_Make services provided to the public available in a digital channel and in a manner that maximizes self-service task or transaction completion_**
- Not require a handwritten signature (“wet signature”) or other in-person identity proofing requirements as a requirement for completing a public-facing form or service without providing the public with an equivalent digital method


I know that we have our forms digitized, but I wonder if ONRR, as a whole, could improve some of the forms and processes related to submitting data and requirements. This is going a bit beyond this goal and probably our scope of work, but I wonder if there are opportunities to provide digital tools to streamline/maximize self service tasks at ONRR.
